component/header.jsx

```diff
@@ -52,38 +157,72 @@ const Version = () => (
   <Div fg="comment">#/bin/nan --hash={HASH.slice(0, 80 - 17)}</Div>
 )
 
+const SideBarFirst = () => (
+  <div class="sidebar__first">
+      <ul>
+          <li>
+              <Link href="/profil">
+                  <div class="user-photo">
+                    <Img source="user.png" alt="photo de profil" />
+                  </div>
+              </Link>
+          </li>
+          <li>
+              <Link href="/">
+              {h(Home)}
```

Comment:
why do you use `h()` method here ? what's the utility here ?
`h()` allow to turn jsx elements into virtual DOM elements according to preact documentation. You use `h(Home)` ? why ?